src/lib/buildWebpackEntries.js

```diff
@@ -6,6 +6,14 @@ function getBasePath () {
   return path.join(process.cwd(), "src", "config", ".entries");
 }
 
+const filterEntries = (entries, groupOfEntries) => {
+  if(groupOfEntries){
```

Comment:
there should be an else block which should throw an error to tell that specified group was not found